corehq/apps/translations/admin.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from django.contrib import admin
+from .models import TransifexOrganization
+from .forms import TransifexOrganizationForm
+
+from corehq.motech.utils import b64_aes_encrypt
+
+
+class TransifexOrganizationAdmin(admin.ModelAdmin):
+    form = TransifexOrganizationForm
+
+    def save_model(self, request, obj, form, change):
+        obj.api_token = b64_aes_encrypt(obj.api_token)
```

Comment:
will this double encrypt if you edit something else on the UI and then save?

Reply:
the view would always show the raw/unencrypted version. So what this would get is the raw version from view each time.
I will double check.